### PR TITLE
Add github links, switch to SCSS

### PR DESCRIPTION
**Pull request by ky28059**
I couldn't find the github link for William, if anyone who knows can add it in afterwards that'd be swell

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,43 +17,45 @@
...
+      <li><a class="link" href="https://github.com/acrantel">Serena Li</a></li>
+      <li><a class="link" href="https://github.com/pinkbluesky">Alina Li</a></li>
+      <li><a class="link" href="https://github.com/omahesh">Om Mahesh</a></li>
+      <li><a class="link" href="https://github.com/BubbyBabur">Roger Fan</a></li>
+      <li><a class="link" href="https://github.com/acd124">Alexander Chang-Davidson</a></li>
+      <li><a class="link" href="https://github.com/ky28059">Kevin Yu</a></li>
+      <li><a class="link" href="https://github.com/saumyasinghal747">Saumya Singhal</a></li>
+      <li><a class="link" href="https://github.com/mg27713">Matthew Grupenhoff</a></li>
+      <li><a class="link" href="https://github.com/ytchang05">Yu-Ting Chang</a></li>
+      <li><a class="link" href="">William Tsin</a></li>
```
Comment (SheepTester) [inline index.html]: Of the members in the organization, @bucketpotato01 isn't listed, so it's probably him

**Review by SheepTester — APPROVED** [review]
Query: How does one build the SCSS should they desire to edit it?

File: index.html (hunk 1 above)
Comment (ky28059) [reply index.html]: Epic, will add!

**Comment by ky28059** [comment]
> Query: How does one build the SCSS should they desire to edit it?

If using WebStorm as an IDE, it will prompt you to set a file watcher which will autocompile non-partial scss files when they are changed (not sure if this needs `sass` to be installed first to work). Otherwise, you can install the sass command line tool with `npm install -g sass` and then either manually compile files or directories to css or use `--watch` to set up a similar file watcher (see the preprocessing section in the sass guide [here](https://sass-lang.com/guide))

**Comment by SheepTester** [comment]
> > Query: How does one build the SCSS should they desire to edit it?
> 
> If using WebStorm as an IDE, it will prompt you to set a file watcher which will autocompile non-partial scss files when they are changed (not sure if this needs `sass` to be installed first to work). Otherwise, you can install the sass command line tool with `npm install -g sass` and then either manually compile files or directories to css or use `--watch` to set up a similar file watcher (see the preprocessing section in the sass guide [here](https://sass-lang.com/guide))

I see, then perhaps we could add the `npm install -g sass` info in the README

File: index.html (hunk 1 above)
Comment (mg27713) [reply index.html]: What if instead of just linking to our githubs we had bio pages that had links to our githubs and homepages?

Comment (SheepTester) [reply index.html]: Yeah, I was thinking that perhaps we could show the list of people as cards, using Github profile pictures as an image and perhaps linking to other social media :eyes:

Comment (ky28059) [reply index.html]: Good idea, I'll try adding that as well.

**Comment by ky28059** [comment]
2 main things the card update needs review on:
- Color of cards: currently the color handling is kind of funky and en peu laid
- Font: I am not good with fonts, and the default font was not great either

**Comment by ky28059** [comment]
Current look
For people who don't want to clone and open locally

**Comment by SheepTester** [comment]
:+1: :D We can add the other links in a separate PR

**Pull request merged**